packages/web-app-files/src/components/SideBar/Shares/FileShares.vue

```diff
@@ -145,6 +173,18 @@ export default {
         : ''
     },
 
+    sharedWithLabel() {
+      return this.$gettext('Shared with')
+    },
+
+    hasSharees() {
+      return this.collaborators_avatar.length > 0
+    },
+
+    sharedWithTooltip() {
+      return this.$gettext('Click to show all')
```

Comment:
```suggestion
      return this.$gettext('Show all invited people')
```
because a) the interaction is not limited to a click and b) the tooltip should give more context than just `all`, but instead what exactly you get when you interact with the button.

Reply:
Also, if `this.showShareesList` is true, you need to return `this.$gettext('Collapse list of invited people')`.